packages/browser/src/node/providers/playwright.ts

```diff
@@ -71,6 +71,18 @@ export class PlaywrightBrowserProvider implements BrowserProvider {
         headless: options.headless,
       } satisfies LaunchOptions
 
+      if (this.ctx.config.inspector.enabled) {
+        // NodeJS equivalent defaults: https://nodejs.org/en/learn/getting-started/debugging#enable-inspector
+        const port = this.ctx.config.inspector.port || 9229
+        const host = this.ctx.config.inspector.host || '127.0.0.1'
+
+        launchOptions.args ||= []
+        launchOptions.args.push(`--remote-debugging-port=${port}`)
```

Comment:
Does this work for all browsers? If not, do browsers just accept invalid args?

Reply:
This works only on Chromium. Webkit seems to hang when this is passed. I'll add checks to `node/config/resolveConfig.ts` for browser+inspect and throw error when unsupported combination is used. 

Firefox has different flag, `--start-debugger-server=${port}`. I wasn't able to get my VSCode stop at correct lines with that one though so I'll leave it out of scope now.